cmd/verifier/main.go

```diff
@@ -101,7 +101,8 @@ func main() {
 	// 1. Fetch latest checkpoint and verify
 	// 2. If old checkpoint is present, verify consistency proof
 	// 3. Write latest checkpoint to file
-	for {
+	ticker := time.NewTicker(*interval)
```

Comment:
Can you add `defer ticket.Stop()` to clean up resources?

Reply:
Sure!